lib/terraforming/resource/iam_group_membership.rb

```diff
@@ -44,7 +44,8 @@ def tfstate
       private
 
       def group_members_of(group)
-        @client.get_group(group_name: group.group_name).users.map { |user| user.user_name }
+        users = @client.get_group(group_name: group.group_name).map { |r| r.users }.flatten
```

Comment:
__[rubocop]__
* Line is too long. [91/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)